harness/features/variable.cloud.test.ts

```diff
@@ -0,0 +1,184 @@
+import { getConnectionStringForProxy, forEachSDK, describeIf, createClient, createUser } from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import nock from 'nock'
+
+jest.setTimeout(1000000)
+
+const scope = getServerScope()
+
+describe('Variable Tests - Cloud', () => {
+    forEachSDK((name) => {
+        let url: string
+        const capabilities: string[] = SDKCapabilities[name]
+        const clientId: string = uuidv4()
+
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)
+            await createClient(url, clientId, 'dvc_server_test_token')
```

Comment:
do you not need to add the option `{ enableCloudBucketing: true }`

Reply:
yep, i'll commit it to the branch